runtime/runtime/src/config.rs

```diff
@@ -267,7 +270,16 @@ pub fn tx_cost(
     // If signer is equals to receiver the receipt will be processed at the same block as this
     // transaction. Otherwise it will processed in the next block and the gas might be inflated.
     let initial_receipt_hop = if transaction.signer_id == transaction.receiver_id { 0 } else { 1 };
-    let minimum_new_receipt_gas = fees.min_receipt_with_function_call_gas();
+    let minimum_new_receipt_gas =
+        if !checked_feature!("stable", DecreaseFunctionCallBaseCost, protocol_version) {
+            fees.min_receipt_with_function_call_gas()
+        } else {
+            // The pessimistic gas pricing is a best-effort limit which can be breached in case of
+            // congestion when receipts are delayed before they execute. Hence there is not much
+            // value to tie this limit to the function call base cost. Making it constant limits
+            // overcharging to 6x, which was the value before the cost increase.
+            4_700_000_000_000 // 4.7TGas.
```

Comment:
Looks like this is not the exact sum of action_function_call send and exec cost. Is this intended?